src/main/java/com/sap/oss/phosphor/fosstars/tool/format/AbstractMarkdownFormatter.java

```diff
@@ -350,14 +332,15 @@ String detailsOf(ScoreValue scoreValue) {
     if (!subScoreValues.isEmpty()) {
       subScoreValues.sort(Collections.reverseOrder(Comparator.comparingDouble(ScoreValue::weight)));
 
-      sb.append(format("This sub-score is based on the following sub-score%s:%n%n",
+      elements.add(Markdown.string(
+          "This sub-score is based on the following sub-score%s:%n%n",
           subScoreValues.size() == 1 ? StringUtils.EMPTY : "s"));
-      sb.append(highLevelDescriptionOf(scoreValue));
-      sb.append("\n");
+
+      elements.add(highLevelDescriptionOf(scoreValue));
     }
 
     if (!featureValues.isEmpty()) {
-      sb.append(format("This sub-score is based on %d feature%s:%n%n",
+      elements.add(Markdown.string("This sub-score is based on %d feature%s:%n%n",
```

Comment:
Make these `Markdown.string` as static import?

Reply:
I thought about it but `Markdown.string` sounds better to me. Let's keep it as-is. If we need to make the lines shorter in the future, we can add a static import.

Reply:
I thought about it but `Markdown.string` sounds better to me. Let's keep it as-is. If we need to make the lines shorter in the future, we can add a static import.